### DISCUSSION TITLE
Cannot use "FormProvider" from "react-hook-forms"

### DISCUSSION THREAD

**Discussion by GabrielGomesBarbosa** [react-hook-form/react-hook-form#2163] [Q&A] [answered]
Im trying to use FormProvider  from "react-hook-forms" but i get error `'FormProvider' is not exported from 'react-hook-form'.`

I just copy like documentation v5 in https://react-hook-form.com/v5/api#useFormContext

**Comment by bluebill1049** [comment, marked as answer]
Thanks! fixed the doc, should be `FormContext`

**Reply by GabrielGomesBarbosa** [reply]
Thanks @bluebill1049, I'm your fan

**Reply by bluebill1049** [reply]
Thank you @GabrielGomesBarbosa for your support 🙏

**Reply by jerrywebie** [reply]
Thank you very much @bluebill1049 . i got the same error and fixed it with `<FormContext>`.

**Reply by bluebill1049** [reply]
No worries ☺️
